### PR TITLE
chore: create basic GitHub action CI

### PR DESCRIPTION
**Pull request by nikku**
This adds a GitHub actions CI pipeline to our build.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,7 @@ jobs:
       before_script:
         - ./tasks/wiredeps
```
Comment (nikku) [inline .travis.yml:10]: We do not do that on GitHub actions right now, should we?

Comment (philippfromme) [reply .travis.yml:10]: Yes, we want the same behavior we previously had on Travis.

Comment (nikku) [reply .travis.yml:10]: O.k. need to add this one then.

**Review by philippfromme — APPROVED** [review]
Had to disable Firefox and PhantonJS on Windows. PhantomJS on Mac doesn't seem to be entirely stable but okay for now.

**Comment by nikku** [comment]
Thanks for approving and looking into the windows issues. I :heart: how easy these kinds of conditional pipelines are:

image

**Pull request merged**